<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HiddenChessClass</class>
 <widget class="QMainWindow" name="HiddenChessClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HiddenChess</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MainMenu" name="mainMenuWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>301</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="RoomCreationWidget" name="roomCreationWidget" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>39</y>
      <width>731</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="JoiningWidget" name="joiningWidget" native="true">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>290</y>
      <width>521</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="GameWidget" name="gameWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>231</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="errLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="errTextBrowser">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
border:none;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1176</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainMenu</class>
   <extends>QWidget</extends>
   <header>mainmenu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RoomCreationWidget</class>
   <extends>QWidget</extends>
   <header>roomcreationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JoiningWidget</class>
   <extends>QWidget</extends>
   <header>joiningwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameWidget</class>
   <extends>QWidget</extends>
   <header>gamewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="HiddenChess.qrc"/>
 </resources>
 <connections/>
</ui>
